README.rst

```diff
@@ -1,254 +1,22 @@
-.. image:: https://cloud.githubusercontent.com/assets/15197846/21435381/96e50466-c89f-11e6-8f0c-95267da389cf.png
+.. image:: https://cloud.githubusercontent.com/assets/5716520/24837754/982064c4-1d3b-11e7-9610-bbdd758c97b5.png
+    :target: https://coala.io/
 
-coala: Linting and fixing code for all languages
-------------------------------------------------
+Visit coala.io for More Information
+-----------------------------------
 
-**coala provides a unified command-line interface for linting and fixing all
-your code, regardless of the programming languages you use.**
+coala provides a unified command-line interface for linting and fixing all
+your code, regardless of the programming languages you use.
 
-With coala, users can create
```

Comment:
By removing high level details like this, you will loose people at https://pypi.python.org/pypi/coala or https://github.com/coala/coala, who will not be motivated by a single sentence, so they wont visit a link in order to obtain a better synopsis.